crates/astria-sequencer/Cargo.toml

```diff
@@ -24,10 +24,10 @@ ed25519-consensus = { workspace = true }
 futures = { workspace = true }
 hex = { workspace = true, features = ["serde"] }
 ibc-types = { workspace = true, features = ["with_serde"] }
-penumbra-component = { workspace = true }
+cnidarium = { workspace = true }
```

Comment:
wonder if this should no longer be a workspace dependency but a sequencer-only dependency

Reply:
yeah, looks like only sequencer uses this now. will update